build.sbt

```diff
@@ -25,6 +26,20 @@ val javaDslApiUrl = s"http://fauna.github.io/faunadb-jvm/$driverVersion/faunadb-
 val javaApiUrl = s"http://fauna.github.io/faunadb-jvm/$driverVersion/faunadb-java/api/"
 val javaAndroidApiUrl = s"http://fauna.github.io/faunadb-jvm/$driverVersion/faunadb-android/api/"
 
+// GPG
+PgpKeys.gpgCommand in Global := "gpg2"
```

Comment:
Oh, I did not, since the CI job is configured to do the whole release process atm. On my local system, `gpg2` does not exist. Should I just omit this line then?